operator/pkg/model/translation/envoy_listener.go

```diff
@@ -55,6 +56,37 @@ func WithProxyProtocol() ListenerMutator {
 	}
 }
 
+func WithXffNumTrustedHops(xff uint32) ListenerMutator {
```

Comment:
I see that we already have a test case in `envoy_listener_test.go` to test the default value for `httpConnectionManager.XffNumTrustedHops`, what about adding another test case to test the mutator with a non-default value?

Reply:
Done.